util/translate_timeline.js

```diff
@@ -26,6 +26,26 @@ parser.addArgument(['-c', '--colorize'], {
 
 const rootDir = 'ui/raidboss/data';
 
+const colorize = (lineText, line, lineSync) => {
```

Comment:
Hmm sorry, this is not what I meant at all.  I was looking for a helper function that takes in a string and a color and optionally turns it into a colorized string, e.g.

```typescript
const colorize = (str, colorStr) => {
  if (args?.colorize !== 'true')
    return str;
  // do some lookup of color names here,
  const colorCode = /* something with colorStr */
  return `${colorCode}${str}${endColorCode}`;
};
```

And then there's only one block for printing out everything that always calls colorize for each replacement, rather than a `if (colorize) { print everything colorized } else { print everything not colorized`.  I want a `print everything colorized (where the helper function to colorize sometimes does it or not)`.

Reply:
ok than tbh i am lost ^^'
far to inexpirienced to do this kind of magic as i would not even know where to start with it than.